tests/foreman/cli/test_activationkey.py

```diff
@@ -619,17 +620,19 @@ def test_positive_usage_limit(module_org, module_location, target_sat):
             'lifecycle-environment-id': env['id'],
             'content-view': new_cv['name'],
             'organization-id': module_org.id,
-            'max-hosts': '1',
+            'max-hosts': max_hosts,
         }
     )
-    with Broker(nick='rhel7', host_class=ContentHost, _count=2) as clients:
+    with Broker(nick='rhel8', host_class=ContentHost, _count=2) as clients:
```

Comment:
Can you use content_hosts fixture here instead ?

Reply:
Maybe? I'll take a look tomorrow...

Reply:
Updated.